packages/embark/src/lib/modules/ganache/index.js

```diff
@@ -0,0 +1,10 @@
+class Ganache {
+  constructor(embark) {
+    embark.events.request('proxy:vm:register', () => {
+      const ganache = require('ganache-cli');
+      return ganache.provider();
```

Comment:
It's weird to me that we don't pass the provider instead, but as long as other things work this way, I'm :+1: with it I guess.

It is, however, a case of a constructor doing work. Not sure how I feel about this. @iurimatias ?

Reply:
It is passing the provider. `ganache.provider()` returns a ganache provider. Unless you meant something else?

As for the constructor doing work, well, it's just registering as a vm provider. Not sure there is a much cleaner way to do it.